### PR TITLE
docs: fix minor formatting issues in user guide

### PR DESCRIPTION
**Pull request by danielleloh**
this ensures consistency in the formatting of the user guide.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1020,18 +1020,18 @@ Some commands require you to include parameters. These parameters are identified
...
+| e\|    | Email                     | 100         | **Format:** local-part@domain<br/> **Constraints for local part:**<br/> • Should only contain alphanumeric characters, and the characters `+`, `_`, `.` and `-`.<br/> • Should not start or end with special characters.<br/> • Should not have two consecutive special characters.<br/> **Constraints for domain:**<br/> • Made up of domain labels separated by periods.<br/> • Must end with a domain label of at least 2 characters long.<br/> • Domain label should consist of alphanumeric characters separated only by singular hyphens, if any. |
```
Comment (jayllo-c) [inline docs/UserGuide.md:1028]: Nice typo find

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1020,18 +1020,18 @@ Some commands require you to include parameters. These parameters are identified
...
+| i\|    | Path of CSV File to Import| -           | Should be the absolute file path of the CSV file without any inverted commas.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (Zer0Legion) [inline docs/UserGuide.md:1030]: Thanks for adding this, makes the constraints much more specific

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1020,18 +1020,18 @@ Some commands require you to include parameters. These parameters are identified
...
+| [s\| ] | Score                     | 7 + 2 decimals | For exam max scores: the input must be a positive integer.<br/> For persons' exam scores: the input must be an integer greater than or equal to zero.                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (jayllo-c) [inline docs/UserGuide.md:1034]: Insightful change, we should learn to be more thorough with our casing.

Comment (Zer0Legion) [inline docs/UserGuide.md:1034]: Consistent formatting!

**Review by Zer0Legion — COMMENTED** [review]
Capitalization within the table seems much more consistent

**Review by jayllo-c — COMMENTED** [review]
Good job with the changes!

File: docs/UserGuide.md (hunk 3 above)
Comment (jayllo-c) [reply docs/UserGuide.md:1034]: 👍

File: docs/UserGuide.md (hunk 2 above)
Comment (jayllo-c) [reply docs/UserGuide.md:1030]: 👍

File: docs/UserGuide.md (hunk 3 above)
Comment (Zer0Legion) [reply docs/UserGuide.md:1034]: 👍

Comment (Zer0Legion) [reply docs/UserGuide.md:1034]: 👍

**Pull request merged**